easybuild/easyconfigs/n/NVSHMEM/NVSHMEM-2.5.0-gompi-2022a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,65 @@
+easyblock = 'ConfigureMake'
+
+name = 'NVSHMEM'
+version = '2.5.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://developer.nvidia.com/nvshmem'
+description = """NVSHMEM is a parallel programming interface based on OpenSHMEM that provides 
+efficient and scalable communication for NVIDIA GPU clusters. NVSHMEM creates a 
+global address space for data that spans the memory of multiple GPUs and can be 
+accessed with fine-grained GPU-initiated operations, CPU-initiated operations, 
+and operations on CUDA streams.
+"""
+
+toolchain = {'name': 'gompi', 'version': '2022a'}
+
+# You need to download the source manully from https://developer.nvidia.com/nvshmem-downloads
```

Comment:
would be nice to use `download_instructions` here

Reply:
Oh nice! I was not aware about the `download_instructions` options. Thanks for pointing that out. :+1: 

Added in https://github.com/easybuilders/easybuild-easyconfigs/pull/16254/commits/c4c1a9b5ba47224454f3d4eb3502fa2e520eed03

Reply:
yeah it's neat, a lot nicer than having to dig through the easyconfigs when you encounter missing sources